scripts/linters/pylint_extensions.py

```diff
@@ -2109,6 +2109,44 @@ def visit_call(self, node):
                     break
 
 
+class DisallowBlankLinesBelowFunctionDefinitionChecker(checkers.BaseChecker):
+    """Custom pylint checker to ensure there are no blank lines
+    below function definition.
+    """
+
+    _implements_ = interfaces.IAstroidChecker
+    name = 'disallow-blank-lines-below-function-definition'
+    priority = -1
+    msgs = {
+        'C0034': (
+            'Please remove the blank line(s) below the function definition.',
+            'remove-blank-lines-below-function-definiton.',
+            'Disallow blank lines below function definition.'
+        ),
+    }
+
+    def visit_functiondef(self, node):
+        """Visits every function and ensures there are no blank lines below
+        function definition.
+
+        Args:
+            node: astroid.scoped_nodes.Function. Node for a function or method
+                definition in the AST.
+        """
+        line_number = node.fromlineno
+        while True:
+            line = linecache.getline(node.root().file, line_number).strip()
+            if re.search(r':', line):
```

Comment:
@DubeySandeep This approach LGTM, but leaving a comment for you to cross check this (as I felt this is the only imp part I would want you to look at when you review). Please look at this [comment](https://github.com/oppia/oppia/pull/11980#issuecomment-801953010) to understand this.